GEOSldas_App/lenkf_j_template.py

```diff
@@ -716,14 +716,24 @@
        set THISDIR = $EXPDIR/output/$EXPDOMAIN/rs/$ENSDIR/Y${{eYEAR}}/M${{eMON}}/
        if (! -e $THISDIR            ) mkdir -p $THISDIR
    
-       set rstf = ${{MODEL}} 
-       if (-f ${{rstf}}${{ENSID}}_internal_checkpoint ) then
-          set tmp_file = $EXPDIR/output/$EXPDOMAIN/rs/$ENSDIR/Y${{eYEAR}}/M${{eMON}}/${{EXPID}}.${{rstf}}_internal_rst.${{eYEAR}}${{eMON}}${{eDAY}}_${{eHour}}${{eMin}}
-          /bin/mv ${{rstf}}${{ENSID}}_internal_checkpoint $tmp_file
-          /bin/rm -f $EXPDIR/input/restart/${{rstf}}${{ENSID}}_internal_rst
-          /bin/ln -rs  $tmp_file $EXPDIR/input/restart/${{rstf}}${{ENSID}}_internal_rst
+       set rstfs = (${{MODEL}} 'landice')
```

Comment:
@weiyuan-jiang: "MODEL" here is the particular flavor of the land (Catchment) model.  Now that we're integrating landice and lake, which include their own "model", I think it would be good to rename "MODEL" to "LANDMODEL" for clarity.  Does this make sense?

Reply:
Yes.  I have changed the name